index.d.ts

```diff
@@ -0,0 +1,19 @@
+declare module 'plural-ru' {
+  import { StyleProp, ViewStyle } from 'react-native';
+
+  type ThreeForms = [string, string, string];
+  type TwoForms = [string, string];
+  type Forms = TwoForms | ThreeForms;
+
+  type GetPlural = (num: numbers, ...forms: Forms) => string;
+
+  const noun: GetPlural;
+  const verb: GetPlural;
```

Comment:
Функция `verb` работает только с 3-мя формами, а тип `GetPlural` позволит использовать её с 2-мя формами. Тут надо что-то вроде:
```suggestion
const verb: (num: number, ...forms: ThreeForms) => string;
```